test/Interop/CxxToSwiftToCxx/bridge-cxx-struct-back-to-cxx-execution.cpp

```diff
@@ -1,9 +1,9 @@
 // RUN: %empty-directory(%t)
```

Comment:
@hyp I'll let you decide if you want to land this PR before fixing this test and the next one. Sorry I did such a terrible job "fixing" these. I couldn't always understand the error, so maybe we can work on them offline.